.github/workflows/build_artifacts.yml

```diff
@@ -18,17 +18,4 @@ jobs:
       env:
         JAVA_OPTS: -Xms512m -Xmx1024m
       run: |
-        # Generate compiler-plugin artifact
-        ./gradlew :compile-plugin:shadowJar
-        #
-        # Generate testing-plugin artifact when running compiler-plugin tests
-        ./gradlew :compile-plugin:test
-        #
-        # Run tests from testing-plugin
-        ./gradlew :testing-plugin:test
-        #
-        # Generate idea-plugin artifact
-        ./gradlew :idea-plugin:build
```

Comment:
Hi! What is the motivation behind have the other plugin tasks running removed?

Reply:
Hi! It was an unnecessary change this morning... :woman_facepalming: